Automattic/wp-calypso
# distinguish between legacy and classic blue

#### Changes proposed in this Pull Request

Adding a separate _Classic Blue_ scheme to distinguish between the old default and the new one. This will allow us to develop the new _Classic Blue_ in parallel. For now I only copied the CSS variable values from _Classic Bright_.

#### Testing instructions

* Go to your account settings and see if there's a _Legacy Blue_ and _Classic Blue_ theme
* _Legacy Blue_ should show the colors as is on `master`

<img width="603" alt="screenshot 2018-12-14 at 10 13 17" src="https://user-images.githubusercontent.com/9202899/49994119-ea84b180-ff88-11e8-854f-4d879a8e462c.png">
